### ISSUE TITLE
Use gcloud for deployment of java apps

### ISSUE THREAD

**Issue by mgenov** [bazelbuild/rules_appengine#116]
As application deployments with appcfg are no longer supported it is valuable gcloud to be used and for java apps to ensure that Java apps could be deployed on GAE. 

Addresses the following issue with appcfg: 
```
Application deployment failed. Message: Deployments using appcfg are no longer supported.
```

**Comment by pmbethe09** [comment]
I agree, we should fix this.  probably won't get to this until January though due to some upcoming deadlines.

**Comment by mgenov** [comment]
Hi @pmbethe09 

Any info about gcloud support?
